.github/workflows/dependency-updates.yml

```diff
@@ -0,0 +1,52 @@
+name: hmrc-frontend dependency updates
+
+on:
+  push:
+    branches:
+      - main
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "50 * * * *"
```

Comment:
Is this ran too frequently?

Reply:
For anyone who comes to this in future: Tom's absolutely right. The goal would be to then reduce the frequency to once per day.